Async/ImportProductProcessor.php

```diff
@@ -33,18 +34,31 @@ class ImportProductProcessor implements MessageProcessorInterface, TopicSubscrib
     /** @var SyncProcessorRegistry */
     private $syncProcessorRegistry;
 
+    /** @var JobStorage */
+    private $jobStorage;
+
+    /**
+     * @param DoctrineHelper $doctrineHelper
```

Comment:
Add --dry-run here https://github.com/oroinc/OroAkeneoBundle/blob/1.6/.travis.yml#L14 please and fix builds